### ISSUE TITLE
icons: Brewery icon

### ISSUE THREAD

**Issue by jpds** [organicmaps/organicmaps#2187]
The brewery icon is slightly odd in that it's a hand that looks like a "stop" sign.

Perhaps it should be changed to a purple "beer" icon? Not sure if that clashes with anything accessibility related.

**Comment by endim8** [comment]
Current Icon
<img width="104" alt="Screenshot 2022-02-25 at 14 58 06" src="https://user-images.githubusercontent.com/26939824/155737015-31b0f6e1-ad08-4e39-b713-2d697f655524.png"> 

it seems to be that all shops with craft=* get the hand icon: 
https://github.com/organicmaps/organicmaps/blob/e0f075efcd0c820bc23bf77a27510a3b6fdfe4e8/data/styles/clear/include/Icons.mapcss#L1502-L1504

i think a modified version of  [beer-m.svg](https://github.com/organicmaps/organicmaps/blob/master/data/styles/clear/style-clear/symbols/beer-m.svg) would be a good replacement

**Comment by jpds** [comment]
This is the icon iD uses within the editor: https://github.com/ideditor/temaki/blob/main/icons/storage_fermenter.svg (though not on the site itself).

**Comment by biodranik** [comment]
It would be great to find something like this:
![image](https://user-images.githubusercontent.com/170263/155958468-dd4c3069-8ad2-4202-9383-8d34821b1aeb.png)

**Comment by endim8** [comment]
Maybe something like this depicting brewing equipment? (this is just an example, ideally it'd be more detailed)
<img width="64" src="https://user-images.githubusercontent.com/26939824/172271044-80f3213a-b65e-4e07-ab1f-8e3f559de2b6.svg">

**Comment by Kazing** [comment]
Hello, I have drafted two icons proposal for brewery POI, which depict the fermenter equipment :

![icon-brewery1](https://user-images.githubusercontent.com/41507326/176518672-f60c7f10-8706-4718-a523-a5c9ddc7d52b.svg) ![icon-brewery2](https://user-images.githubusercontent.com/41507326/176518675-2ef23314-878a-4336-a882-1711fe08ed80.svg)

Which one seems most appropriate?

They are based on those two icons :
* https://www.flaticon.com/free-icon/brewery_7837642#
* https://www.flaticon.com/free-icon/brewery_3590681#

**Comment by biodranik** [comment]
How to guess that these icons are related to beer?

**Comment by Kazing** [comment]
They are depicting the brewery equipment, I think it's a good way to signal a brewery.

Other solutions could be : 
* beer glass: but similar to actual pub icon
* hop (https://www.flaticon.com/free-icon/hop_3347268): but I am not sure the general public will recognize hop
* barrel: from my point of view it's more related to wine, but it could be easier to relate this to beer for the general public
* beer bottles: but similar to actual alcohol shop icon

**Comment by dvdmrtnz** [comment]
We could make the symbol as a beer glass inside a little factory outline like this:

![](https://user-images.githubusercontent.com/47610359/185881100-32af5099-81c3-4f20-a8d1-275d84c752ac.png)

**Comment by biodranik** [comment]
A barrel looks better and clearer )

**Comment by dvdmrtnz** [comment]
Okay let's see. Maybe a picture of a barrel like this?

![](https://user-images.githubusercontent.com/47610359/186225683-041cfa2d-bbd2-4c33-8e45-ca5ec5eb4d0b.png)

**Comment by dvdmrtnz** [comment]
I made some alternatives simplifying the previous design: 

1: ![](https://user-images.githubusercontent.com/47610359/186621332-ebf35513-09d0-4ff1-867f-7e81457f9072.png) ![](https://user-images.githubusercontent.com/47610359/186622141-738d00b7-e9b3-4953-8220-1f0600f37ee0.png)

2: ![](https://user-images.githubusercontent.com/47610359/186621344-5bf12b49-2c5c-4982-abaa-d23d3c661e1b.png) ![](https://user-images.githubusercontent.com/47610359/186622161-b0fbd023-b70b-493d-9ae0-f5bb6b832618.png)

3: ![](https://user-images.githubusercontent.com/47610359/186621353-cb93d91a-d7f4-4a59-a4d0-8cb64257e471.png) ![](https://user-images.githubusercontent.com/47610359/186622179-21b5119a-999b-4a66-9cb1-a4868c80c357.png)

**Comment by biodranik** [comment]
3 is no.
1 and 2 are not too clear, the first question is "What is there? A dot? What does it mean?"

**Comment by not-available-username** [comment]
please don't chose an icon with outline like the barrels above, this style is less readable and the consistency of icons is low enough on OM.

A simplified version of this one in my opinion could be good https://github.com/organicmaps/organicmaps/issues/2187#issuecomment-1170401853, maybe something similar?

![1](https://user-images.githubusercontent.com/92886370/186855247-749946c2-265e-47f8-8337-97e209e3e4e2.svg)
![2](https://user-images.githubusercontent.com/92886370/186855261-5d038c6b-0089-4358-aa0b-0030e8b21617.svg)

**Comment by dvdmrtnz** [comment]
> 3 is no. 
1 and 2 are not too clear, the first question is "What is there? A dot? What does it mean?"

Here is a different variation of the barrel icon. Maybe you like this one better?

![](https://user-images.githubusercontent.com/47610359/186861215-dc7ac458-f43c-45b0-abc0-5683fe7c8cf7.png) ![](https://user-images.githubusercontent.com/47610359/186861521-03b9d679-be09-499f-8387-3f61bcbb50cc.png)

**Comment by biodranik** [comment]
> please don't choose an icon with outline like the barrels above, this _outline only_ style is less readable and the consistency of icons is low enough on OM.
> 
> A simplified version of this one in my opinion could be good [#2187 (comment)](https://github.com/organicmaps/organicmaps/issues/2187#issuecomment-1170401853), maybe something similar?
> 
> ![1](https://user-images.githubusercontent.com/92886370/186855247-749946c2-265e-47f8-8337-97e209e3e4e2.svg) ![2](https://user-images.githubusercontent.com/92886370/186855261-5d038c6b-0089-4358-aa0b-0030e8b21617.svg)

Biolab? Garbage cleaners? Toothpaste? That's hardly recognizable.

**Comment by biodranik** [comment]
A barrel looks good (although with a faucet would be even better).

**Comment by dvdmrtnz** [comment]
I tried to add a faucet, the icons are quite small so it's a bit hard to see it, though. What do you think?

![](https://user-images.githubusercontent.com/47610359/187084941-0afc38b6-d544-4e86-84bb-913124c50f96.png) ![](https://user-images.githubusercontent.com/47610359/187084849-763f059f-d382-4962-af19-02ca6b30d4c7.png)

**Comment by biodranik** [comment]
That's better only if zoomed, but for the map, it looks too detailed. Maybe try something like this (simplified a bit)?
![image](https://user-images.githubusercontent.com/170263/187092366-b1c83a30-fd3b-4784-b7f9-d5c9c2f5cd03.png)

**Comment by dvdmrtnz** [comment]
Maybe like this?

![](https://user-images.githubusercontent.com/47610359/187093850-085bb01f-8203-417c-9223-25716a8704de.png)
![](https://user-images.githubusercontent.com/47610359/187093889-ccd248c8-9b0a-4a6e-800e-5b3787e3d01a.png)

**Comment by biodranik** [comment]
Still not recognisable :( Maybe remove vertical lines, move the faucet a bit lower, and add a horizontal stand line?

**Comment by dvdmrtnz** [comment]
Are you sure about removing the vertical lines? I think they make the barrel more recognisable. Here are a couple variations:

1: ![](https://user-images.githubusercontent.com/47610359/187095207-37ef7fe1-39f8-4ccf-a897-f1cddc35099f.png) ![](https://user-images.githubusercontent.com/47610359/187095246-33a66da1-91ff-4e2b-9fe6-c1c032abbb67.png)


2: ![](https://user-images.githubusercontent.com/47610359/187095209-4c4bed88-fe55-44b3-8247-03fe62658550.png) ![](https://user-images.githubusercontent.com/47610359/187095232-6b9b61f5-0294-40c3-9792-8b5e523bd298.png)

**Comment by biodranik** [comment]
Not sure ) It doesn't look good, you're right...

**Comment by dvdmrtnz** [comment]
> > 3 is no.
> > 1 and 2 are not too clear, the first question is "What is there? A dot? What does it mean?"
> 
> Here is a different variation of the barrel icon. Maybe you like this one better?
> 
> ![](https://user-images.githubusercontent.com/47610359/186861215-dc7ac458-f43c-45b0-abc0-5683fe7c8cf7.png) ![](https://user-images.githubusercontent.com/47610359/186861521-03b9d679-be09-499f-8387-3f61bcbb50cc.png)

Looking back a couple of months later, I think the best looking one is the simple barrel. I think we should add this one to the app for now, and then if someone has a better looking one they can make a new PR. 

@biodranik If you agree, I can make the PR.

**Comment by biodranik** [comment]
I agree, the last barrel looks better than anything else.
